### ISSUE TITLE
User testing suggestion: notification styles

### ISSUE THREAD

**Issue by ajacobs21e** [openshift/origin#6411]
Users mentioned that notifications that some notifications were hard to spot and some were too large. 

I mocked up two notification styles based on PatternFly. For notifications where the user would like to see status information but does not need to take an action, a toast notification is used:
![notification - toast](https://cloud.githubusercontent.com/assets/6415742/11902673/ad1e39b0-a581-11e5-8f43-cea5e0e46f14.png)
https://www.patternfly.org/patterns/toast-notifications/
This notification will disappear after 8 seconds.

For notifications where crucial information is displayed or the user has to (or should) take an action, an inline notification is used:
![notification - inline](https://cloud.githubusercontent.com/assets/6415742/11902730/0111836a-a582-11e5-9d9f-ef650273e4fc.png)
https://www.patternfly.org/patterns/inline-notifications/
This notification remains on the screen until dismissed by the user.

**Comment by ajacobs21e** [comment]
I did user testing at Customer Convergence and here are the results: 
[OpenShift Notifications.pdf](https://github.com/openshift/origin/files/123164/OpenShift.Notifications.pdf)

tl;dr: 
![openshift notifications](https://cloud.githubusercontent.com/assets/6415742/12918856/626b10ca-cf0f-11e5-8d92-41f2aa0bcc32.png)

https://www.patternfly.org/patterns/inline-notifications/
https://www.patternfly.org/patterns/toast-notifications/

**Comment by jwforres** [comment]
Moving to trello for tracking purposes https://trello.com/c/6WVJdgjL
